tests/run-make/jobserver-error/cannot_open_fd.stderr

```diff
@@ -0,0 +1,6 @@
+warning: failed to connect to jobserver from environment variable `MAKEFLAGS="--jobserver-auth=3,3"`: cannot open file descriptor 3 from the jobserver environment variable value: Bad file descriptor (os error 9)
+         
+         the build environment is likely misconfigured
```

Comment:
I meant attaching this as a note to the warning, as notes are usually attached to primary diagnostics, not just adding newlines.
You can add a missing interface for this to `EarlyErrorHandler` if the current interface is not sufficient.